git_machete/cmd.py

```diff
@@ -349,38 +349,38 @@ def get_root_of(b: str) -> str:
             warn("no branches satisfying the criteria. Try moving the value of `--checked-out-since` further to the past.")
             return
 
-        for b in excluding(non_root_fixed_branches, stale_non_root_fixed_branches):
-            u = self.__infer_upstream(b, condition=lambda candidate: get_root_of(candidate) != b and candidate not in stale_non_root_fixed_branches, reject_reason_message="choosing this candidate would form a cycle in the resulting graph or the candidate is a stale branch")
-            if u:
-                debug("discover_tree()", f"inferred upstream of {b} is {u}, attaching {b} as a child of {u}\n")
-                self.up_branch[b] = u
-                root_of[b] = u
-                if u in self.__down_branches:
-                    self.__down_branches[u].append(b)
+        for branch in excluding(non_root_fixed_branches, stale_non_root_fixed_branches):
+            upstream = self.__infer_upstream(branch, condition=lambda candidate: get_root_of(candidate) != branch and candidate not in stale_non_root_fixed_branches, reject_reason_message="choosing this candidate would form a cycle in the resulting graph or the candidate is a stale branch")
```

Comment:
Pls split this super-long line while you're here

Reply:
Do you mean using `\`?

Reply:
Yeah, pls use `\` if needed... but I think it shouldn't be needed, just put each param into separate line

Reply:
Hmmm pls share the error message from your machine 🤔

Reply:
`            upstream = self.__infer_upstream(branch, condition=lambda
                candidate: get_root_of(
                candidate) != branch and candidate not in stale_non_root_fixed_branches,
                                             reject_reason_message="choosing this candidate would form a cycle in the resulting graph or the candidate is a stale branch")
`

Autoformat like this gives me `E128`, `E122`and `E126`

Reply:
Pls push the autoformatted line and we'll see how to fix that on this PR

Reply:
pushed

Reply:
Pls apply this diff:

```diff
diff --git a/git_machete/cmd.py b/git_machete/cmd.py
index 9b94d02..a7cdd1d 100644
--- a/git_machete/cmd.py
+++ b/git_machete/cmd.py
@@ -350,10 +350,10 @@ class MacheteClient:
             return
 
         for branch in excluding(non_root_fixed_branches, stale_non_root_fixed_branches):
-            upstream = self.__infer_upstream(branch, condition=lambda
-                candidate: get_root_of(
-                candidate) != branch and candidate not in stale_non_root_fixed_branches,
-                                             reject_reason_message="choosing this candidate would form a cycle in the resulting graph or the candidate is a stale branch")
+            upstream = self.__infer_upstream(
+                branch,
+                condition=lambda candidate: get_root_of(candidate) != branch and candidate not in stale_non_root_fixed_branches,
+                reject_reason_message="choosing this candidate would form a cycle in the resulting graph or the candidate is a stale branch")
             if upstream:
                 debug("discover_tree()", f"inferred upstream of {branch} is {upstream}, attaching {branch} as a child of {upstream}\n")
                 self.up_branch[branch] = upstream
```

and we can merge 🎉

Reply:
Thanks! Pushed.